examples/with-custom-compress-encodings/next.config.js

```diff
@@ -0,0 +1,22 @@
+var zlib = require('zlib')
+var iltorb = require('iltorb')
+
+module.exports = {
+  // Returns a map of compression streams for the types of encodings you want to support
+  // Here's a list of common encoding types: https://goo.gl/ke7zOK
+
+  // The first listed encoding has the higher priority over others.
+  //  In this case, first it'll try to serve the `br` version if the browser supports it.
+  //  Otherwise, it'll serve the gzipped version.
+  compress: {
+    br: function () {
```

Comment:
Isn't is the same as right now except it should be `br: iltorb.compressStream()`?

If you are looking at ES2015 features, I didn't use them because we don't transpile the `next.config.js`.